### DISCUSSION TITLE
validator operable using Java 11?

### DISCUSSION THREAD

**Discussion by Dnoack49** [INSPIRE-MIF/helpdesk-validator#782] [General]
Dear all,

we currently use a local instance of the validator (but no Docker-image) and our machines will switch to java 11 soon. A year after [this question ](https://github.com/INSPIRE-MIF/helpdesk-validator/issues/503) regarding Java 11 was raised, we would like to know if the status is the same or if there are any plans to move it to the new java-version and if it will be runnable using JDK11/Tomcat9?

Kind regards

**Comment by jenriquesoriano** [comment]
Dear @Dnoack49 ,

thank you very much for your question.
We are currently working on the incorporation of a new version of the ETF to the INSPIRE reference validator.  
This version will be compatible with Java 11.
As soon as it is available, a specific instance will be published for testing by the community and the different artifacts will be made available for deployment.
We will keep you informed of the evolution of this process.

Best regards,

**Reply by Dnoack49** [reply]
Dear @jenriquesoriano ,

since the issue [Migrate to Java 11 #71]( https://github.com/etf-validator/governance/issues/71) has been closed in August I assume that with the [Version 2.1.0](https://github.com/etf-validator/etf-webapp/releases/tag/2.1.0) a local validator-instance (no docker image) can be implemented using using JDK11/Tomcat9 ?

**Reply by Dnoack49** [reply]
Dear @jenriquesoriano ,

I would appreciate a feedback on my question above.

kind regards

**Reply by Dnoack49** [reply]
Dear @jenriquesoriano ,

I would appreciate a feedback on my question above.

kind regards

**Reply by jenriquesoriano** [reply]
Dear @Dnoack49,

as you correctly point, there should be no problem deploying the ETF version 2.1 using Java 11 and Tomcat 9. 
We are currently working in the production of release of the INSPIRE Reference Validator using this set-up, adapting it to the expected behaviour.
As soon as we have it ready for testing, we will let you know.

Best,
